docker/specter-desktop/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3
+
+RUN apt-get update && apt-get install -y libusb-1.0-0-dev libudev-dev
+RUN pip3 install cryptoadvance.specter
```

Comment:
To be honest, for the official docker-image, i really like the idea to install a specifc version via pip.
More about that in a general comment.

Reply:
This could be a build `ARG`, where by default it copies the repo files into the image (good for developers, people who want unreleased version), but if a specific version is passed it uses `pip` to install it.

Reply:
Oh, that would be cool! But we can have it as a separate PR, i'd say.